### ISSUE TITLE
Vertical bar of pixels appearing on the left side of the recordings

### ISSUE THREAD

**Issue by Soulflier** [begeekmyfriend/yasea#569]
After increasing output resolution from 640x360 to 1280x720, there is a visible bar of pixels on the left side of recording, which looks, like it should appear on the right side of the recording. It's clearly visible in the [video](https://youtu.be/hkhUiVlZxpo). 

To reproduce it in the demo, in the MainActivity the line
`mPublisher.setOutputResolution(360, 640);`
should be simply swapped for
`mPublisher.setOutputResolution(720, 1280);`

The problem appears only in recordings and streams. The preview visible in the app appears normal.
The problem appears only when recording horizontally.
The problem appears both in front and back cameras.

Changing preview resolution to match the output resolution doesn't fix anything.
The video I linked has been made on Samsung Galaxy S7, although I have been able to reproduce the issue on a couple of devices I have access to.

Am I missing anything? I've tried to find the source of the problem for a couple of days with no success so far.

**Comment by begeekmyfriend** [comment]
I see. It happens on any resolution horizontally when recording. I will try to find something in [SrsMp4Muxer.java](https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/java/net/ossrs/yasea/SrsMp4Muxer.java)

**Comment by Soulflier** [comment]
The issue also appears in the stream attempts I made so far. Based on that, if it's a problem in [SrsMp4Muxer.java](https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/java/net/ossrs/yasea/SrsMp4Muxer.java), then it has to be somehow replicated in [SrsFlvMuxer.java](https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/java/net/ossrs/yasea/SrsFlvMuxer.java)

**Comment by begeekmyfriend** [comment]
It need to mux H.264 and AAC and encapsulates in some media container.

**Comment by begeekmyfriend** [comment]
I do not think it is the issue of muxing. In fact you can use `ffplay` to find out that it still reappear in live broadcasting. I am afraid the problem might happen in process of `libyuv` module. There may be some bug in [libenc.cc](https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/cpp/libenc/libenc.cc)
